doc/cascadia/profiles.schema.json

```diff
@@ -531,6 +531,11 @@
           "minimum": 1,
           "type": "integer"
         },
+		"fontWeight": {
```

Comment:
TABS BRO. I TOLD YOU ABOUT TABS.

![image](https://user-images.githubusercontent.com/18221333/82482074-2465bb80-9a8b-11ea-8110-60598f2f2450.png)